app/models/liquid_tags/user_subscription_tag.rb

```diff
@@ -0,0 +1,9 @@
+module LiquidTags
+  class UserSubscriptionTag < ApplicationRecord
```

Comment:
`Rolify` only works with model resources 😢 . I chose this setup to avoid naming conflicts and organize these into one directory.

Reply:
I kinda want to set aside these stubs in their own folder. Work for another day though.